packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -1083,27 +1083,33 @@ describe('network stubbing', { retries: 2 }, function () {
     })
 
     context('body parsing', function () {
-      it('automatically parses JSON request bodies', function () {
-        const p = Promise.defer()
+      [
+        ['application/json', '{"foo":"bar"}'],
+        ['application/vnd.api+json', '{}'],
+      ].forEach(([contentType, expectedBody]) => {
+        it(`automatically parses ${contentType} request bodies`, function () {
+          const p = Promise.defer()
 
-        cy.intercept('/post-only', (req) => {
-          expect(req.body).to.deep.eq({ foo: 'bar' })
+          cy.intercept('/post-only', (req) => {
+            expect(req.headers['content-type']).to.eq(contentType)
+            expect(req.body).to.deep.eq({ foo: 'bar' })
```

Comment:
but not `expectedBody`?

Reply:
Yes, `expectedBody` is for the expected response body only - Express will not auto-parse the POSTed JSON with the content-type `application/vnd.api+json`, so that is why they differ